src/Lachain.Core/Blockchain/Error/OperatingError.cs

```diff
@@ -27,5 +27,6 @@ public enum OperatingError : byte
         BlockGasOverflow,
         Underpriced,
         ValueOverflow,
+        BadTransaction,
```

Comment:
I think it is better to make more narrow error type here,  smth like DuplicatedTransaction or smth like this

Reply:
Ok